app/form_objects/provider_visa_form.rb

```diff
@@ -0,0 +1,25 @@
+class ProviderVisaForm
+  include ActiveModel::Model
+
+  attr_reader :can_sponsor_skilled_worker_visa, :can_sponsor_student_visa
+
+  validates :can_sponsor_skilled_worker_visa, inclusion: { in: [true, false], message: "Select whether your provider can sponsor skilled worker visas" }
+  validates :can_sponsor_student_visa, inclusion: { in: [true, false], message: "Select whether your provider can sponsor student visas" }
```

Comment:
Minor: Can you change the order of validations to match the order of the fields on the page?